examples/webgl_loader_sea3d_morph.html

```diff
@@ -187,6 +182,9 @@
 
 				requestAnimationFrame( animate );
 
+				// Update SEA3D Animations
+				THREE.SEA3D.AnimationHandler.update( delta );
```

Comment:
I see `optionalRoot` in `mixer.clipAction()` I'll test as soon as possible. Thanks!